tests/test_pe_reports.py

```diff
@@ -89,6 +90,7 @@ def test_bad_log_level():
             "2021-01-01",
             "input/",
             "output/",
+            "--db-creds-file=/secrests/database_creds.yml",
```

Comment:
Same question as above:  Is there any value in adding `--db-creds-file` here?  It seems to me like there is not.

Reply:
Resolved in b980e9e